geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/AlterRegionCommandDUnitTest.java

```diff
@@ -100,6 +115,73 @@ public void testAlterRegionResetCacheListeners() throws IOException {
     });
   }
 
+  @Test
+  public void alterEntryIdleTimeExpiration() {
+    gfsh.executeAndAssertThat(
+        "create region --name=regionA --type=REPLICATE --entry-idle-time-expiration=10 --enable-statistics")
+        .statusIsSuccess();
+
+    server1.invoke(() -> {
+      Region region = ClusterStartupRule.getCache().getRegion("regionA");
+      ExpirationAttributes expiry = region.getAttributes().getEntryIdleTimeout();
+      assertThat(expiry.getTimeout()).isEqualTo(10);
+      assertThat(expiry.getAction()).isEqualTo(ExpirationAction.INVALIDATE);
+    });
+
+    gfsh.executeAndAssertThat(
+        "alter region --name=regionA --entry-idle-time-expiration-action=DESTROY")
+        .statusIsSuccess();
+    server1.invoke(() -> {
+      Region region = ClusterStartupRule.getCache().getRegion("regionA");
+      ExpirationAttributes expiry = region.getAttributes().getEntryIdleTimeout();
+      assertThat(expiry.getTimeout()).isEqualTo(10);
+      assertThat(expiry.getAction()).isEqualTo(ExpirationAction.DESTROY);
+    });
+
+    gfsh.executeAndAssertThat("alter region --name=regionA --entry-idle-time-expiration=5")
+        .statusIsSuccess();
+    server1.invoke(() -> {
+      Region region = ClusterStartupRule.getCache().getRegion("regionA");
+      ExpirationAttributes expiry = region.getAttributes().getEntryIdleTimeout();
+      assertThat(expiry.getTimeout()).isEqualTo(5);
+      assertThat(expiry.getAction()).isEqualTo(ExpirationAction.DESTROY);
+    });
+  }
+
+
+  @Test
```

Comment:
We should have a test that spins up a region that does not have statistics enabled, and then tries to add an expiry condition (which of course fails, since statistics are required).

And/or one where a region starts with statistics enabled but without an expiry condition, and then we add a expiry condition to the region.

Reply:
test added